src/functions/enclose.js

```diff
@@ -34,6 +35,40 @@ const htmlBuilder = (group, options) => {
         img.height = options.fontMetrics().defaultRuleThickness / scale;
         imgShift = -0.5 * options.fontMetrics().xHeight;
 
+    } else if (label === "angl" || label === "lcroof") {
+        const vlineThickness = calculateSize({number: 0.4, unit: "pt"}, options);
+        let hlineThickness = vlineThickness;
+
+        if (label === "lcroof") {
+            hlineThickness = calculateSize({number: 0.6, unit: "pt"}, options);
+        }
+
+        // Put some space before inner (but under the angle)
+        inner.classes.push("actangle-pad");
+
+        const parenDepth = 0.25; //in em, the depth of a parenthesis
+        inner.depth = Math.max(inner.depth, parenDepth);
+
+        imgShift += parenDepth;
+
+        // Height of the angle computation :
+        // 1. Take inner's height + depth of a parenthesis,
+        //    because we only want to extend this far down.
+        const totalInnerHeight = inner.height + parenDepth;
+        // 2. Account for the hline (for thicker hlines especially)
+        // 3. Add 15/24rd (hey, magic number) of the parenthesis' descender's
```

Comment:
Could you add comments about these constants, including `0.4pt`, `0.6pt`, and `0.25` above?

Reply:
See #baf4c6e